### PR TITLE
[ADD] open_academy: Added course demo data T#59081

### PR DESCRIPTION
**Pull request by JesusValdez96**
Creation of demo data for the course model
Link to exercise: https://www.odoo.com/documentation/15.0/developer/howtos/backend.html#data-files

### CODE REVIEW

**Comment by JesusValdez96** [comment]
Please review @andreaarce

**Comment by andreaarce** [comment]
@CarmenMiranda Please help me with the review

File: open_academy/__manifest__.py [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
+    "license": "LGPL-3",
 
-    'category': 'Uncategorized',
-    'version': '15.0.1.0.0',
+    "category": "Uncategorized",
+    "version": "15.0.1.0.0",
 
-    'depends': ['base'],
+    "depends": [
+        "base"
```
Comment (CarmenMiranda) [inline open_academy/__manifest__.py:15]: I recommend adding a comma here, so when another dependency is added just one line is added and not one added and another is modified

**Review by CarmenMiranda — CHANGES_REQUESTED** [review]
I just add  a comment, and I going to recommend or do 2 commits, the first one with the readme and the fix with the pre-commit, and the second commit with the demo, or the other option is to add those changes in the description of the commit

@andreaarce What do you think about the commits?

File: open_academy/__manifest__.py (hunk 1 above)
Comment (andreaarce) [reply open_academy/__manifest__.py:15]: I prefer one commit with the changes in the description

File: open_academy/__manifest__.py [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
+    "author": "Jesús Valdez, Vauxoo",
+    "website": "https://github.com/JesusValdez96/open_academy",
 
-    'author': "Jesús Valdez [Vauxoo]",
-    'website': "https://github.com/JesusValdez96/open_academy",
+    "license": "LGPL-3",
 
-    'category': 'Uncategorized',
-    'version': '15.0.1.0.0',
+    "category": "Uncategorized",
```
Comment (andreaarce) [inline open_academy/__manifest__.py:11]: Change the category to Customizations

File: open_academy/__manifest__.py [hunk 3]
```diff
@@ -1,21 +1,25 @@
...
 
-    'depends': ['base'],
+    "depends": [
+        "base"
+    ],
 
-    'data': [
-        'security/ir.model.access.csv'
+    "data": [
+        "security/ir.model.access.csv"
```
Comment (andreaarce) [inline open_academy/__manifest__.py:19]: Also the same comment than @CarmenMiranda please add a comma to the final line

File: open_academy/__manifest__.py [hunk 4]
```diff
@@ -1,21 +1,25 @@
...
     ],
 
-    'demo': [],
+    "demo": [
+        "demo/course_demo.xml"
```
Comment (andreaarce) [inline open_academy/__manifest__.py:23]: Add a comma

File: open_academy/security/ir.model.access.csv [hunk 5]
```diff
@@ -1,2 +1,2 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_open_academy_open_academy,open_academy.open_academy,model_open_academy_open_academy,base.group_user,1,1,1,1
\ No newline at end of file
+access_open_academy_open_academy,open_academy.open_academy,model_open_academy_open_academy,base.group_user,1,1,1,1
```
Comment (andreaarce) [inline open_academy/security/ir.model.access.csv:2]: This line has no effect on the project the open_academy model doesn't exist please replace to the course model and change all the related references

Comment (CarmenMiranda) [reply open_academy/security/ir.model.access.csv:2]: Oh, I didn't know that the model didn't exist 🙈

**Comment by JesusValdez96** [comment]
Please review again @andreaarce, @CarmenMiranda

**Review by CarmenMiranda — APPROVED** [review]
@andreaarce Technically LGTM 👍🏼

**Comment by CarmenMiranda** [comment]
@JesusValdez96 Just remove the draft state of the merge 😄

**Comment by JesusValdez96** [comment]
Removed. Thanks! @CarmenMiranda

**Pull request merged**